imports/plugins/core/orders/server/no-meteor/resolvers/Query/orders.js

```diff
@@ -0,0 +1,34 @@
+import { getPaginatedResponse, wasFieldRequested } from "@reactioncommerce/reaction-graphql-utils";
+import { decodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/shop";
+
+/**
+ * @name Query/orders
+ * @method
+ * @memberof Order/GraphQL
+ * @summary Get an order by its reference ID
+ * @param {Object} parentResult - unused
+ * @param {ConnectionArgs} args - An object of all arguments that were sent by the client
+ * @param {String} args.filter - filter by _id, referenceId, email and name
+ * @param {String} args.orderStatus - workflow status to limit search results
+ * @param {String} args.shopIds - shop IDs to check for orders from
+ * @param {Object} context - An object containing the per-request state
+ * @param {Object} info Info about the GraphQL request
+ * @returns {Promise<Object>|undefined} An Order object
+ */
+export default async function orders(parentResult, args, context, info) {
+  const { filter, orderStatus, shopIds: opaqueShopIds, ...connectionArgs } = args;
+
+  const shopIds = opaqueShopIds && opaqueShopIds.map(decodeShopOpaqueId);
+
+  const query = await context.queries.orders(context, {
+    filter,
+    orderStatus,
+    shopIds
+  });
+
+  return getPaginatedResponse(query, connectionArgs, {
```

Comment:
Will this support offset pagination? or are we planning on using cursor based pagination in the orders table?

Reply:
Planning for the offset based pagination. From what I've seen from the implementation, you add the offset field to the GQL schema and use `getPaginatedResponse` as normal. All the logic is in `getPaginatedResponse` to do offset base pagination.

Reply:
OK, great!